Packs/Claroty/ReleaseNotes/1_0_18.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Claroty
+- Updated the Docker image to: *demisto/python3:3.10.6.33415*.
```

Comment:
Let's also mention the rest of the updates here (should be according to our [templates](https://xsoar.pan.dev/docs/documentation/release-notes)):
Add a new field to the configuration form for including/excluding resolved alerts in the response's payload
Add default query filters to the assets requests
Add default query filters to the alerts requests